cluster/examples/kubernetes/ceph/common.yaml

```
@@ -304,6 +305,14 @@ rules:
       - list
       - get
       - watch
+  - apiGroups:
+      - "storage.k8s.io"
```

nit: quotes shouldn't be needed
```suggestion
      - storage.k8s.io
```